components/cli/pkg/commands/image/build.go

```diff
@@ -71,33 +70,33 @@ func RunBuild(cli cli.Cli, tag string, fileName string) error {
 		}); err != nil {
 		return err
 	}
-	// Execute ballerina build in temporary executable bal file
+	// Execute ballerina build in temporary executable bal file.
 	if err = cli.ExecuteTask("Executing ballerina build", "Failed to execute ballerina build",
 		"", func() error {
 			err := executeTempBalFile(cli.BalExecutor(), tempBuildFileName, iName)
 			return err
 		}); err != nil {
 		return err
 	}
-	// Generate metadata
+	// Generate metadata.
 	if err = cli.ExecuteTask("Generating metadata", "Failed to generate metadata",
 		"", func() error {
 			err := generateMetaData(cli, parsedCellImage, projectDir)
 			return err
 		}); err != nil {
 		return err
 	}
-	// Create the artifacts zip file
+	// Create the artifacts zip file.
 	artifactsZip := parsedCellImage.ImageName + cellImageExt
-	if err = cli.ExecuteTask("Creating cell image zip file", "Failed to create temporary baf file",
+	if err = cli.ExecuteTask("Creating cell image zip file", "Failed to image zip",
```

Comment:
s/Failed to image zip/failed to create image zip/

Reply:
Updated the error message to "Failed to create the image zip"